src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -167,3 +178,61 @@ const applicantRelationshipEventMap = {
 export function recordApplicantRelationship({ formData: { onBehalfOf } }) {
   return <UserInteractionRecorder eventRecorder={recordEvent} selectedValue={onBehalfOf} trackingEventMap={applicantRelationshipEventMap}></UserInteractionRecorder>;
 }
+
+/**
+ * A helper that takes data from the SchoolSelectField back end and transforms
+ * it to a valid format as specified by the FEEDBACK-TOOL's
+ * educationDetails.school.address schema.
+ *
+ * @param {*} _ - Object with address fields
+ * @returns {Object} An Object that passes the FEEDBACK-TOOL's educationDetails.
+ * school.address schema.
+ */
+export function transformSearchToolAddress({ address1, address2, address3, city, country, state, zip }) {
+  const isDomesticAddress = country === 'USA';
```

Comment:
It seems inconsistent to use an abbreviation for USA only in the school address country enum list, would it make sense to create a ticket to convert this enum to the full name? If so, we could use the country value below on line 195 rather than a magic string.

Reply:
you might be misreading this (or I'm misunderstanding)? the search tool returns 'USA' and we're transforming it to 'United States'

Reply:
Thanks for clarifying, that makes sense.